### ISSUE TITLE
QA does not calculate F1 score result , may I know how to fix it?

### ISSUE THREAD

**Issue by chaoyanghe** [FedML-AI/FedNLP#10]
root:epoch = 1, batch_idx = 2164/5520, loss = 0.6054576635360718
INFO:root:epoch = 1, batch_idx = 2165/5520, loss = 0.4317961633205414
INFO:root:epoch = 1, batch_idx = 2166/5520, loss = 1.390831470489502
INFO:root:epoch = 1, batch_idx = 2167/5520, loss = 1.07370924949646
INFO:root:epoch = 1, batch_idx = 2168/5520, loss = 1.1164920330047607
INFO:root:epoch = 1, batch_idx = 2169/5520, loss = 0.6192945241928101
INFO:root:epoch = 1, batch_idx = 2170/5520, loss = 0.7042073607444763
INFO:root:epoch = 1, batch_idx = 2171/5520, loss = 0.702218770980835
INFO:root:epoch = 1, batch_idx = 2172/5520, loss = 0.47233307361602783
INFO:root:epoch = 1, batch_idx = 2173/5520, loss = 0.547944188117981
INFO:root:epoch = 1, batch_idx = 2174/5520, loss = 0.703558087348938
INFO:root:epoch = 1, batch_idx = 2175/5520, loss = 0.793656587600708
INFO:root:epoch = 1, batch_idx = 2176/5520, loss = 0.790333092212677
INFO:root:epoch = 1, batch_idx = 2177/5520, loss = 0.5816390514373779
INFO:root:epoch = 1, batch_idx = 2178/5520, loss = 0.9623005986213684
INFO:root:epoch = 1, batch_idx = 2179/5520, loss = 0.6054102182388306
INFO:root:cached_features_file = cache_dir/cached_dev_bert_256_34726
INFO:examples.question_answering.question_answering_model: Features loaded from cache at cache_dir/cached_dev_bert_256_34726
Running Evaluation: 100%|██████████| 2203/2203 [03:30<00:00, 10.46it/s]
INFO:root:{}
INFO:examples.**question_answering.question_answering_model:{'correct': 15510, 'similar': 14006, 'incorrect': 5210, 'eval_loss': -7.354878266291677}**
INFO:root:epoch = 1, batch_idx = 2180/5520, loss = 0.5322230458259583
INFO:root:epoch = 1, batch_idx = 2181/5520, loss = 0.5274325609207153
INFO:root:epoch = 1, batch_idx = 2182/5520, loss = 0.5773954391479492
INFO:root:epoch = 1, batch_idx = 2183/5520, loss = 0.8108208775520325
